### PR TITLE
Overlay blur fix + slightly less white background

### PR DESCRIPTION
**Pull request by MisterMakerNL**
Goes from this:
image
to this:
image
@Syres916

### CODE REVIEW

**Comment by FEA-eng** [comment]
Nice. So this should fix both #15609 and #15749, right?

**Comment by MisterMakerNL** [comment]
> Nice. So this should fix both #15609 and #15749, right?

yeah, thnx for the ping on those. Saves me some searching around!

**Comment by MisterMakerNL** [comment]
The white background has to be between #ffffff and #f0f0f0 , ended up with #f7f7f7.

**Comment by obelisk79** [comment]
What does that look like with a solid behind it now?

**Comment by MisterMakerNL** [comment]
> What does that look like with a solid behind it now?
chernobyl-meme-not-great-not-terrible-1555278232

File: CMakeSettings.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+      "variables": [
+        {
+          "name": "FREECAD_LIBPACK_DIR",
+          "value": "C:/freecad_libs/LibPack",
+          "type": "PATH"
+        }
+      ]
+    }
+  ]
+}
```
Comment (AjinkyaDahale) [inline CMakeSettings.json:22]: This entire file should not be a part of this PR. Please remove this from the commit.

Comment (MisterMakerNL) [reply CMakeSettings.json]: Totally right I missed that! Is deleted.

Comment (AjinkyaDahale) [reply CMakeSettings.json]: Thank you. But as I mentioned earlier, remove this from the history itself, not just by adding a commit that does the deletion. You can do that by squashing the deletion commit with the one that added the file.

Comment (MisterMakerNL) [reply CMakeSettings.json]: I am on windows on github desktop. I see no buttons, please don't make me go to console ;)

**Pull request merged**